specification/containerservice/resource-manager/readme.go.md

```diff
@@ -67,6 +68,16 @@ batch:
   - tag: package-2017-07
 ```
 
+### Tag: package-2022-09 and go
+
+These settings apply only when `--package-2022-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2022-09' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2022-09-01/$(namespace)
+```
```

Comment:
We do not recommend to update the configurations here because this is for track 1 Go SDK and it has been deprecated.
Also the pipeline has been turned off therefore these configurations will not be validated.

Reply:
Get it, removed.